genesyscloud/resource_genesyscloud_journey_segment.go

```diff
@@ -0,0 +1,623 @@
+package genesyscloud
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"regexp"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/mypurecloud/platform-client-sdk-go/v72/platformclientv2"
```

Comment:
I've just updated to v74 of the SDK, can you rebase and update this please.

Reply:
Main merged, updating new code too.